src/view/UserMgt.java

```diff
@@ -511,7 +520,7 @@ private String getUserLevel() {
         }
     }
     
-    private void adminWindowLinkMouseClicked(java.awt.event.MouseEvent evt) {//GEN-FIRST:event_adminWindowLinkMouseClicked
+    private void adminWindowLinkMouseClicked(java.awt.event.MouseEvent evt) throws SQLException {//GEN-FIRST:event_adminWindowLinkMouseClicked
```

Comment:
Why did you add `throws SQLException` , yet we are not dealing with any Sql stuffs here?!

Reply:
It's because, in simple terms, in Java, the way exceptions work is like this: sometimes you want to catch errors when the code is not executed as expected or if there is something that is wrong. That's why, usually we surround our code with "try catch" statements. Alternatively, instead of using "try catch" statements that can make your code look hard to read and less clean, you can throw an exception at the declaration of a method.
Example: 
```
public User getUser() throws UserNotFoundException {
//write some code.
}
```
However, this approach comes with a trade-off, each time you call the method that throws an exception, you will have to throw it again in the method that is making the call 

```
public void addUser() throws UserNotFoundException {     //This method will also throw the exception
User user = getUser(); //this trows a UserNotFoundException
//write some code
}
```

This approach will ensure that all the methods that are hierarchically linked to the method that throws the exception, will also throw the exception.